modules/core/src/main/java/org/apache/fluo/core/oracle/Stamp.java

```diff
@@ -15,11 +15,18 @@
 
 package org.apache.fluo.core.oracle;
 
+import org.apache.fluo.accumulo.util.ColumnConstants;
+import com.google.common.base.Preconditions;
+
 public class Stamp {
   private final long txStamp;
   private final long gcStamp;
 
   Stamp(long stamp, long gcStamp) {
+	long b= stamp & ColumnConstants.PREFIX_MASK ;
```

Comment:
The formatting of this still file is incorrect.  I suspect one of the following happened.

 * After running `mvn compile`, didn't commit and push this file.  Personally I like to run `mvn compile` followed by `git status` to see what the formatter changed.
 * Your IDE has auto-formatting turned on and it reformatted the file after mvn did.  If you are interested I can help you find instructions for setting up Fluo formatting in Eclipse and IntelliJ.

I can fix this when I merge this in or if you would like to fix it, I can wait for that.  Just let me know.

Reply:
@keith-turner I am available now and would be interested in learning. I am using Eclipse.

Reply:
We have instructions at the end of this page : http://fluo.apache.org/how-to-contribute/.  However I just looked at them and saw that they are incomplete.  We need to fix that page.  Anyway, here are the instructions.

There are two things to do: configure organizing imports and configure code formatting.

Configuring organizing imports is on the website.  But it should be `Window -> Preferences -> Java -> Code Style -> Organize Imports` instead of `Window -> Preferences -> Java -> Organize Imports`.

The instructions for configuring the formatter are not even on the page.  To do that, try :

  * Go to `Window -> Preferences -> Java -> Code Style -> Formatter`
  * Click Import and import `<local Fluo source dir>/contrib/fluo-eclipse-style.xml`
  * Either configure all projects in eclipse to use this style or just the Fluo projects.

Reply:
I opened apache/fluo-website#86 to improve the instructions.

Reply:
@keith-turner Thanks so much for detailed instructions. Just imported fluo code style in my eclipse IDE. Would you want me to do mvn compile and do git status and push it again ?

Reply:
sounds good. will do that now.